webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -43,7 +43,7 @@ const HwPropertiesCard = ({ isExpandedGlobal, hostDetails }) => {
   const coreSocket = facts?.['cpu::core(s)_per_socket'];
   const reportedFacts = propsToCamelCase(hostDetails?.reported_data || {});
   const totalDisks = reportedFacts?.disksTotal;
-  const memory = facts?.['dmi::memory::maximum_capacity'];
+  const memory = facts?.['dmi::memory::size'];
```

Comment:
I think, this is wrong and should stay at maximum_capacity:

sub-man facts:
![image](https://github.com/Katello/katello/assets/25485845/81c2b934-d394-46bd-bfd2-8be226faf058)


puppet facts:

![image](https://github.com/Katello/katello/assets/25485845/01180fe0-a062-4e3d-8907-b02db7ae1c43)

Reply:
What about having a max capacity field and then a size field?

Reply:
If the user does not have puppet facts, then I guess we just only show max capacity?

Reply:
the question is, is it about "hardware properties" as the headline describes or hardware resources the OS can handle?
There are some other facts, e.g. from subscription manager.
Maybe just use what is already there:
https://github.com/theforeman/foreman/blob/31956e5267cc74b7e06eec73042a24505ba45c6f/app/services/katello/rhsm_fact_parser.rb#L107

or 
https://github.com/Katello/katello/blob/506c5282e72d1c4dcbd5a579d189641b0ec84f4b/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js#L159

this is what users expect I guess.